docs/core/settings/variables.md

```diff
@@ -356,3 +356,45 @@ Examples:
 # If %{user} is "testuser", return "INVALID". Otherwise return %{user} uppercased.
 %{user | if ("=", "testuser, "invalid", user) | upper }
 ```
+
+## Cryptography support
+
+### Parameters
+
+| Key                | Value                                                                                                       |
+| ------------------ | ----------------------------------------------------------------------------------------------------------- |
+| key                | Encryption key, if no salt is given, must be hex encoded and match the expected length of chosen algorithm. |
+| iv                 | Initialization vector, provide if no salt is given and algorithm requires one. Must be hex encoded and match the expected length of chosen algorithm. |
```

Comment:
Would be good to have a similar sentence structure as `key`. Maybe `Initialization vector, if no salt is given and algorithm requires one.` Or maybe these both could be rewritten somehow. I couldn't really think of very clear and compact explanation.